modules/lotamePanoramaIdSystem.js

```diff
@@ -16,8 +16,9 @@ const MODULE_NAME = 'lotamePanoramaId';
 const NINE_MONTHS_MS = 23328000 * 1000;
 const DAYS_TO_CACHE = 7;
 const DAY_MS = 60 * 60 * 24 * 1000;
+const GVLID = 95;
 
-export const storage = getStorageManager(null, MODULE_NAME);
+export const storage = getStorageManager(GVLID, MODULE_NAME);
```

Comment:
Let's make sure we confirm with prebid that:
1 - These changes will have 0 effect in the case where gpdr enforcement module is not present
2 - If gdpr enforcement module is present and we don't have consent based on their rules, execution will stop inside getStorageManager and the rest of our code won't run